src/Components/Notifications/NotificationsList.tsx

```diff
@@ -22,6 +22,7 @@ import { NotificationsListPlaceholder } from "./NotificationsListPlaceholder"
 import { useNotificationsContext } from "Components/Notifications/useNotificationsContext"
 import { NotificationListMode } from "Components/Notifications/NotificationsTabs"
 import { useRouter } from "System/Router/useRouter"
+import { __internal__useMatchMedia } from "Utils/Hooks/useMatchMedia"
```

Comment:
It's generally not advised to use [ __internal__useMatchMedia ](https://github.com/artsy/force/blob/cfd8aa9666d4dd3820b8fa962181b666c23ba1fc/src/Utils/Hooks/useMatchMedia.ts#L4-L6) , but in this case I don't think we can use [frensel](https://github.com/artsy/fresnel?tab=readme-ov-file#server-side-rendering-ssr-usage) so we can move on with this current approach